### PR TITLE
[SW-1495] Expose Number of Trees in SW MOJO

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Comment by mn-mikke** [comment]
Need to add some additional tests

File: scoring/build.gradle [hunk 1]
```diff
@@ -10,6 +10,7 @@ dependencies {
     compile "ai.h2o:mojo2-runtime-api:${mojoPipelineVersion}"
     compile "ai.h2o:mojo2-runtime-impl:${mojoPipelineVersion}"
     compile "ai.h2o:h2o-genmodel:${h2oVersion}"
+    compile "ai.h2o:h2o-genmodel-ext-xgboost:${h2oVersion}"
```
Comment (jakubhava) [inline scoring/build.gradle]: Good fix!

File: doc/src/site/sphinx/deployment/load_mojo.rst [hunk 2]
```diff
@@ -105,4 +105,23 @@ or in Python:
 
     from pysparkling.ml import *
     settings = H2OMOJOSettings(convertUnknownCategoricalLevelsToNa = True, convertInvalidNumbersToNa = True)
-    model = H2OMOJOModel.createFromMojo("prostate.mojo", settings)
\ No newline at end of file
+    model = H2OMOJOModel.createFromMojo("prostate.mojo", settings)
```
Comment (jakubhava) [inline doc/src/site/sphinx/deployment/load_mojo.rst]: qq: In Pysparkling, we return the correct type, right?

Comment (mn-mikke) [reply doc/src/site/sphinx/deployment/load_mojo.rst]: Thanks to duck typing in Python it shouldn't matter if we call `H2OMOJOModel.createFromMojo` or `H2OTreeBasedSupervisedMOJOModel.createFromMojo`. I've added some tests [here](https://github.com/h2oai/sparkling-water/pull/1693/files#diff-17206c76b471d879b3ad5ad38e2283e8R165-R177)

**Pull request merged**